src/main/java/com/rarchives/ripme/ripper/rippers/ImagefapRipper.java

```diff
@@ -308,7 +308,7 @@ private long checkRateLimit() {
         } else if(duration / 1000 <  300){
             logger.debug("Rate limit: " + (rateLimitFiveMinutes - callsMade) + " calls remaining for first 5 minute mark.");
         } else if(duration / 1000 <  3600){
-            logger.debug("Rate limit: " + (RATE_LIMIT_HOUR - callsMade) + " calls remaining for first hour mark.");
+            logger.debug("Rate limit: " + (rateLimitHour - callsMade) + " calls remaining for first hour mark.");
```

Comment:
meh, this is fine but also change for the sake of change. I think we probably should change this so that none of these constants are using local variables or magic numbers. (Move towards all magic constants being named constants like RATE_LIMIT_HOUR and not assigned to local variables. Unless, in this case, these delays are expected to change duration under some circumstances.)